src/eoa/eoaHelpers.ts

```
@@ -40,3 +42,73 @@ export async function submitEOAWithdrawalRequest({
+
+      // Execute batch deposit for this batch
+      const tx = await batchDepositContract.batchDeposit(depositData, {
+        value: totalValue,
+      });
+
+      const receipt = await tx.wait();
+      if (receipt?.hash) {
+        txHashes.push(receipt.hash);
+      }
+
+    }
```

ESLint

[error] Block must not be padded by blank lines.

(padded-blocks)